### DISCUSSION TITLE
The indent behavior of `term list` is different from `list`

### DISCUSSION THREAD

**Discussion by astrojhgu** [typst/typst#3360] [Q&A] [answered]
I notice that a `term list` behavior is just like a set of paragraphs. So that the first item does not affected by the intent setting, while begin with the 2nd item, the indent works.

Here is my minimum working code:
![image](https://github.com/typst/typst/assets/2651231/ae2d978c-333b-4f4e-a0e7-3a11b0d88459)
I need the first team in a term list aligned with other items.

Complete code:
```typst
#set par(first-line-indent: 2em)
#h(2em) This is a paragraph, which is manually indented

This is a paragraph automatically indented

#set text(fill:green)
+ This list is correctly indented
+ This list is correctly indented
+ This list is correctly indented

#set text(fill:black)
This is the 3rd paragraph automatically indented

#set text(fill:red)
/ This : line is not indented correctly $<-$ Note this line
/ This : line is correctly indented
/ This : line is correctly indented
```

**Comment by PgBiel** [comment, marked as answer]
This should be fixed in the next update: https://github.com/typst/typst/pull/3325

**Reply by astrojhgu** [reply]
Thanks for the update.
